projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -862,10 +862,10 @@ public static function get_default_label_from_type( $type ) {
 				$str = __( 'Message', 'jetpack-forms' );
 				break;
 			case 'checkbox-multiple':
-				$str = __( 'Choose several', 'jetpack-forms' );
+				$str = __( 'Choose several options', 'jetpack-forms' );
```

Comment:
The label defined in the editor ([here](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/src/blocks/contact-form/components/jetpack-field-single-choice/index.js#L37)) doesn't seem to be available in the [server render callback](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/src/contact-form/class-contact-form-plugin.php#L458), since it's a default value. That's probably why labels are defined again in this `switch`.